--- sata.csr	2015-06-03 15:09:26.000000000 -0700
+++ cn88xx/sata.csr	2015-06-03 15:10:35.000000000 -0700
@@ -63,33 +63,12 @@
       - name: UAHC_INTRQ_IP
         value: 0x000
         attributes:
-          chip_pass: "t88>1.1;t83"
-          pib_int_field: "INTRQ_IP"
-        description: See SATA()_UAHC_GBL_IS[IPS] and SATA()_UAHC_P0_IS.
-
-      - name: UAHC_INTRQ_IP
-        value: 0x000
-        attributes:
-          chip_pass: "t88<=1.1;!t83"
-          intsn_level_sensitive: "True"
           pib_int_field: "INTRQ_IP"
         description: See SATA()_UAHC_GBL_IS[IPS] and SATA()_UAHC_P0_IS.
 
       - name: UCTL_INTSTAT
         value: 0x001
         attributes:
-          chip_pass: "t88>1.1;t83"
-          exempt_pass1_address_change: "True"
-          pib_rtl_module: "sata"
-        description: |
-          See interrupt clears SATA(0..15)_UCTL_INTSTAT, interrupt sets
-          SATA(0..15)_UCTL_INTSTAT_W1S, enable clears
-          SATA(0..15)_UCTL_INTENA_W1C, and enable sets SATA(0..15)_UCTL_INTENA_W1S.
-
-      - name: UCTL_INTSTAT
-        value: 0x004
-        attributes:
-          chip_pass: "t88<=1.1;!t83"
           exempt_pass1_address_change: "True"
           pib_rtl_module: "sata"
         description: |
